diff --git a/src/Storage/EntryModel.php b/src/Storage/EntryModel.php
index 7acaa6101dfed07bd271338cf59b0c646c156dec..50b47a4c97527edfa4a4cb3354e81a27f8dcdc38 100644
--- a/src/Storage/EntryModel.php
+++ b/src/Storage/EntryModel.php
@@ -58,9 +58,9 @@ class EntryModel extends Model
     /**
      * Scope the query for the given query options.
      *
-     * @param \Illuminate\Database\Eloquent\Builder $query
-     * @param string $type
-     * @param \Laravel\Telescope\Storage\EntryQueryOptions $options
+     * @param  \Illuminate\Database\Eloquent\Builder  $query
+     * @param  string  $type
+     * @param  \Laravel\Telescope\Storage\EntryQueryOptions  $options
      * @return \Illuminate\Database\Eloquent\Builder
      */
     public function scopeWithTelescopeOptions($query, $type, EntryQueryOptions $options)
@@ -78,8 +78,8 @@ public function scopeWithTelescopeOptions($query, $type, EntryQueryOptions $opti
     /**
      * Scope the query for the given type.
      *
-     * @param \Illuminate\Database\Eloquent\Builder $query
-     * @param string $type
+     * @param  \Illuminate\Database\Eloquent\Builder  $query
+     * @param  string  $type
      * @return $this
      */
     protected function whereType($query, $type)
@@ -94,8 +94,8 @@ protected function whereType($query, $type)
     /**
      * Scope the query for the given batch ID.
      *
-     * @param \Illuminate\Database\Eloquent\Builder $query
-     * @param \Laravel\Telescope\Storage\EntryQueryOptions $options
+     * @param  \Illuminate\Database\Eloquent\Builder  $query
+     * @param  \Laravel\Telescope\Storage\EntryQueryOptions  $options
      * @return $this
      */
     protected function whereBatchId($query, EntryQueryOptions $options)
@@ -107,12 +107,11 @@ protected function whereBatchId($query, EntryQueryOptions $options)
         return $this;
     }
 
-
     /**
      * Scope the query for the given type.
      *
-     * @param \Illuminate\Database\Eloquent\Builder $query
-     * @param \Laravel\Telescope\Storage\EntryQueryOptions $options
+     * @param  \Illuminate\Database\Eloquent\Builder  $query
+     * @param  \Laravel\Telescope\Storage\EntryQueryOptions  $options
      * @return $this
      */
     protected function whereTag($query, EntryQueryOptions $options)
@@ -123,6 +122,7 @@ protected function whereTag($query, EntryQueryOptions $options)
                     return $q->where('tag', $tag);
                 });
             }
+
             return $query;
         });
 
@@ -142,8 +142,8 @@ public function tags()
     /**
      * Scope the query for the given type.
      *
-     * @param \Illuminate\Database\Eloquent\Builder $query
-     * @param \Laravel\Telescope\Storage\EntryQueryOptions $options
+     * @param  \Illuminate\Database\Eloquent\Builder  $query
+     * @param  \Laravel\Telescope\Storage\EntryQueryOptions  $options
      * @return $this
      */
     protected function whereFamilyHash($query, EntryQueryOptions $options)
@@ -158,8 +158,8 @@ protected function whereFamilyHash($query, EntryQueryOptions $options)
     /**
      * Scope the query for the given pagination options.
      *
-     * @param \Illuminate\Database\Eloquent\Builder $query
-     * @param \Laravel\Telescope\Storage\EntryQueryOptions $options
+     * @param  \Illuminate\Database\Eloquent\Builder  $query
+     * @param  \Laravel\Telescope\Storage\EntryQueryOptions  $options
      * @return $this
      */
     protected function whereBeforeSequence($query, EntryQueryOptions $options)
@@ -174,8 +174,8 @@ protected function whereBeforeSequence($query, EntryQueryOptions $options)
     /**
      * Scope the query for the given display options.
      *
-     * @param \Illuminate\Database\Eloquent\Builder $query
-     * @param \Laravel\Telescope\Storage\EntryQueryOptions $options
+     * @param  \Illuminate\Database\Eloquent\Builder  $query
+     * @param  \Laravel\Telescope\Storage\EntryQueryOptions  $options
      * @return $this
      */
     protected function filter($query, EntryQueryOptions $options)
diff --git a/src/Storage/EntryTagModel.php b/src/Storage/EntryTagModel.php
index e6fefb565311bb27da7dea4c5b9b4b4b49408792..3ddd82de49915999d4df4d9134357cb2f0bb8022 100644
--- a/src/Storage/EntryTagModel.php
+++ b/src/Storage/EntryTagModel.php
@@ -2,7 +2,6 @@
 
 namespace Laravel\Telescope\Storage;
 
-use Illuminate\Database\Eloquent\Factories\HasFactory;
 use Illuminate\Database\Eloquent\Model;
 
 class EntryTagModel extends Model
